fields/index.js

```diff
@@ -25,7 +25,10 @@ module.exports = {
     }
   },
   'dob': {
-    validate: ['british-date'],
+    validate: ['british-date', 'past', {
+      type: 'since',
+      arguments: [require('moment')('1/1/1990', 'DD/MM/YYYY')]
```

Comment:
So far our best guess at the correct date would be '01/07/2009'. But GRO should be able to confirm.